docs/docsite/rst/collections_tech_preview.rst

```diff
@@ -152,7 +152,7 @@ TBD. Expect tests for the collection itself, including Molecule files, to reside
 Creating Collections
 ====================
 
-This is currently is a work in progress. We created the `Mazer <https://galaxy.ansible.com/docs/mazer/index.html>`_ command line tool
+This is currently is a work in progress. We created the `Mazer <https://galaxy.ansible.com/docs/mazer/>`_ command line tool
```

Comment:
This link differs from https://github.com/ansible/ansible/pull/58669/files#diff-7358ca26fe2af8805e3efdc0a0f4f0deR234

@bcoca ^

Reply:
as per above, setting both to 'non devel' should be fine at this point